cumulus/client/consensus/aura/src/collators/mod.rs

```diff
@@ -126,50 +127,33 @@ async fn async_backing_params(
 	}
 }
 
-// Return all the cores assigned to the para at the provided relay parent.
+// Return all the cores assigned to the para at the provided relay parent, using the claim queue
+// offset. This assumes the relay chain runtime supports the claimqueue runtime API.
```

Comment:
I think we can not remove this remark about the relay chain runtime, since it is really supported everywhere now.